scripts/partition_manager/partition_manager.rst

```diff
@@ -8,9 +8,8 @@ When creating an application that requires child images (for example, a bootload
 
 See :ref:`ug_multi_image` and *Building and Configuring multiple images* in :ref:`zephyr:application` for more information about multi-image builds.
 
-By default, all |NCS| samples that require multiple images are configured to use the Partition Manager.
-If you prefer to build and program each image separately and disable the Partition Manager, configure your parent image (the application) to skip building the child images.
-Note that the Partition Manager is invoked only in multi-image builds.
+Partition Manager will automatically be activated for all images that
```

Comment:
Isn't this still just true for NCS samples, as was previously written?

Reply:
If hello_world is built for 10090ns, then Partition Manager is invoked.

Reply:
It's not just for NCS samples, no.

Reply:
@hakonfam @SebastianBoe your answers seem contradictory to me.

Sebastian says PM is active "whenever there are multiple images enabled", but surely hello_world is not enabling multiple images.

What am I missing?

Reply:
It could be, if a ns board is used.